sys-devel/gcc/gcc-9.3.0-r1.ebuild

```diff
@@ -1,13 +1,13 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="6"
```

Comment:
It would be nice to have comments on why we need to keep `EAPI=6` for Flatcar.

Reply:
The original ebuild files quickly are buried in Gentoo's upstream git history. I would prefer to have a copy of them in the commit that introduces them.
Any modifications and comments should come in a second commit - otherwise I wouldn't even expect comments to be in the ebuild file.